### ISSUE TITLE
Segmentation training for curved lines

### ISSUE THREAD

**Issue by theo-roulet** [mittagessen/kraken#744]
Hi kraken users and devs. I'm looking for some advice on the way to train a segmentation model able to handle slightly curved lines, as on the image below.

<img width="400" alt="ES_screenshot" src="https://github.com/user-attachments/assets/dcaec6dd-6a35-496c-bb04-33e26734847a" />

I've tried to train on a 30, a 40 and then a 60 pages set. I made sure that at least 25% then, 50% of my training pages would contain this type of curved lines (in reality they're present on 10% of the pages i will transcribe).
I've set the learning rate at 0.0001 and raised the minimum number of epochs to 50, 100 and even 150 epochs. And I've tried to finetune the blla model downloaded form this repositiry as well as to train from scratch (even though if I remember the documentation correctly, it does the same ?). 
But so far nothing works : my curved lines are still spitted. More generally the model continues to miss a few initial or final letters. 

What do you think ? Is it because my ketos segtrain parameters are not fit for this task ? (I have access to a computation server and I use cuda acceleration). Do I simply need more training data ? Or is such as situation (a minority of curved lines on the whole) too difficult to handle with Kraken and I should rather try to unwarp all my images beforehand ? 
Thanks for any suggestions :)
